Kickstarter-iOS/Features/Messages/Controller/MessagesViewController.swift

```diff
@@ -87,6 +91,19 @@ internal final class MessagesViewController: UITableViewController {
     self.viewModel.outputs.goToBacking
       .observeForControllerAction()
       .observeValues { [weak self] params in self?.goToBacking(with: params) }
+
+    self.viewModel.outputs.userBlocked
+      .observeForUI()
+      .observeValues { [weak self] success in
+
+        if success {
+          self?.messageBannerViewController?
+            .showBanner(with: .success, message: "This user has been successfully blocked")
+        } else {
+          self?.messageBannerViewController?
+            .showBanner(with: .error, message: "Your request did not go through. Try again.")
```

Comment:
Nit: Add a TODO here and for the other banner messages to update them once translated strings are ready